v2/pkg/runner/options.go

```diff
@@ -314,3 +314,153 @@ func userHomeDir() string {
 	}
 	return usr.HomeDir
 }
+
+// ParseUserOptions parses the command line flags provided by a user
+func ParseUserOptions(options *Options) *Options {
+	logutil.DisableDefaultLogger()
+	// Seed default random number generator
+	rand.Seed(time.Now().UnixNano())
+
+	// Migrate config to provider config
+	if fileutil.FileExists(defaultConfigLocation) && !fileutil.FileExists(defaultProviderConfigLocation) {
+		gologger.Info().Msgf("Detected old %s config file, trying to migrate providers to %s\n", defaultConfigLocation, defaultProviderConfigLocation)
+		if err := migrateToProviderConfig(defaultConfigLocation, defaultProviderConfigLocation); err != nil {
+			gologger.Warning().Msgf("Could not migrate providers from existing config %s to provider config %s: %s\n", defaultConfigLocation, defaultProviderConfigLocation, err)
+		} else {
+			// cleanup the existing config file post migration
+			_ = os.Remove(defaultConfigLocation)
+			gologger.Info().Msgf("Migration successful from %s to %s.\n", defaultConfigLocation, defaultProviderConfigLocation)
+		}
+	}
+
+	//options := &Options{}
```

Comment:
It's a good idea to eliminate any code that's been commented out and that you're not planning on using.

Reply:
I have eliminated the commented code